### ISSUE TITLE
Unable to import after installing via Pip

### ISSUE THREAD

**Issue by tobiasvl** [mdzhang/goodreads-api-client-python#17]
```tvl@rocinante:~ $ python3
Python 3.6.5 (default, Mar 29 2018, 18:20:46) 
[GCC 8.0.1 20180317 (Red Hat 8.0.1-0.19)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import goodreads_api_client
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.6/site-packages/goodreads_api_client/__init__.py", line 10, in <module>
    from goodreads_api_client.client import Client
  File "/usr/local/lib/python3.6/site-packages/goodreads_api_client/client.py", line 5, in <module>
    import goodreads_api_client.resources as resources
ModuleNotFoundError: No module named 'goodreads_api_client.resources'
```

**Comment by mdzhang** [comment]
Dupe of #9. Fixed in #14, just now released in `0.1.0.dev4`

**Comment by mdzhang** [comment]
Thanks for noting the issue. Please close of `0.1.0.dev4` fixes it for you!
